src/components/UserList/Row.js

```diff
@@ -1,32 +1,53 @@
 import React, { Component } from 'react';
 import {
+  Alert,
+  Dimensions,
   Image,
+  Platform,
   StyleSheet,
   Text,
   TouchableWithoutFeedback,
+  TouchableHighlight,
   View,
 } from 'react-native';
 import { Actions } from 'react-native-router-flux';
+import LinearGradient from 'react-native-linear-gradient';
 
 class Row extends Component {
 
   constructor(props) {
     super(props);
 
+    // Current location and skills will be changed with real data later.
     this.state = {
       profileImage: '',
       name: '',
       job: '',
-      education: '',
+      position: 'employer',
+      currentLocation: 'San Jose, CA, USA',
+      skills: ['# Motivation', '# prefered_skill', '# freelance'],
     };
   }
 
   componentWillMount() {
+    let job = this.state.job;
+    let position = this.state.position;
+    let skills = this.state.skills;
+
+    if (this.props.dataSource.work.length > 0) {
+      job = 'at ' + this.props.dataSource.work[0].employer.name;
```

Comment:
This looks good for now. But in the future, we can probably make this i18n-ready.


Typically, string formulation like this is done in a separate function, so that internationalization becomes easy. 

For example:

string constructJobLine(string locale, string jobTitle, string companyName) {
  if (jobTitle == "") {
    return companyName;
  } else if (companyName == "") {
    return jobTitle;
  }
  if (locale == "en-US") { 
    return jobTitle + " at " + companyName;
  }
  //TODO(ethkim): Complete all other i18n strings for other languages.
}